internal/dataplane/parser/translate_httproute.go

```diff
@@ -412,6 +453,47 @@ func generatePluginsFromHTTPRouteFilters(filters []gatewayv1beta1.HTTPRouteFilte
 	return kongPlugins
 }
 
+func generateRequestRedirectKongPlugin(modifier *gatewayv1beta1.HTTPRequestRedirectFilter, p string) []kong.Plugin {
```

Comment:
Normally I, wouldn't suggest to add a godoc but in case of a param called `p` :) it's either this or naming param a bit more descriptively. WDYT?